src/Command/ContentHubEntityScanOrphaned.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+namespace Acquia\Console\ContentHub\Command;
+
+use Acquia\Console\ContentHub\Command\Helpers\DrushWrapper;
+use Acquia\Console\Helpers\Command\PlatformCmdOutputFormatterTrait;
+use Acquia\Console\Helpers\PlatformCommandExecutioner;
+use EclipseGc\CommonConsole\Platform\PlatformCommandTrait;
+use EclipseGc\CommonConsole\PlatformCommandInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * Provides command to scan for orphaned entities in Content Hub.
+ */
+class ContentHubEntityScanOrphaned extends Command implements PlatformCommandInterface {
+
+  use PlatformCommandTrait;
+  use PlatformCmdOutputFormatterTrait;
```

Comment:
I think we can remove this trait. Are we using it?

Reply:
Yes, later as `extractAndAggregateJson()`